app/utils.py

```diff
@@ -183,3 +183,8 @@ def get_logo_url(logo_file):
 
 def get_document_url(lang: str, path: str):
     return f'https://{current_app.config["DOCUMENTATION_DOMAIN"]}/{lang}/{path}'
+
+
+def strip_keys_from_personalisation_if_send_attach(personalisation):
```

Comment:
needed to avoid attached files floating errors when validating template variables

Reply:
Not sure what's the recommended way between `type(v) is dict` and `isinstance(v, dict)`. Same?

Reply:
Usually the pythonic way is more use of 'is'. But they're pretty much the same. 

I don't moving it back to post_notifications, I think I used utils.py as it seemed like a place for one-offs like this, I'm good either way.